### PR TITLE
init commit for react week1

### PR DESCRIPTION
**Pull request by DarinkaGM**

### CODE REVIEW

File: react/week1/my-app-w1/src/App.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    h1 > Darinka 's to-do list</h1> <
+    /header> <
+    Radio.Group value = {
+      selectionType
+    } >
+    <
+    /Radio.Group> <
+    Table rowSelection = {
+      {
+        type: selectionType
```
Comment (senner008) [inline react/week1/my-app-w1/src/App.js:96]: If I look at the documentation for the the "antd" library you have used, then I an see that you are missing some logic to get the checkbox feature working.

To get the checkboxes working I had to do the following

Copied this logic from the documentation:
https://ant.design/components/table/
```
 const rowSelection = {
    onChange: (selectedRowKeys, selectedRows) => {
      console.log(`selectedRowKeys: ${selectedRowKeys}`, 'selectedRows: ', selectedRows);
    },
    getCheckboxProps: (record) => ({
      disabled: record.name === 'Disabled User', // Column configuration not to be checked
      name: record.name,
    }),
  };
```

The rowSelection property then has the above object methods:

```
<Table
        rowSelection={{
          type: 'checkbox',
          ...rowSelection
        }}
        columns={columns}
        dataSource={data}
      />

```

Then you are miissing a key on each object:

```
const data = [
  {
    Id: '1',
    key: '1',
    Task: 'React Homework',
    Deadline: 'May-22nd-2021',
  },
  {
    Id: '2',
    key: '2',
    Task: 'UI assingments',
    Deadline: 'June-15th-2021',
  }
];
```

See if you can get it working

**Review by senner008 — APPROVED** [review]
Hi

Take a look at the comments. See if you can fix some of the issues. 

Also, try and see if you can implement the same logic without a library.
We will be expanding on the todo idea later, so its best to have a good understanding of how to build this kind of logic yourself.

File: react/week1/my-app-w1/src/App.js [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    Deadline: 'Agust-2021',
+  },
+
+
+];
+
+
+const App = () => {
+  const [selectionType] = useState('checkbox');
+  return ( <
```
[the remark this answers is not in the document]
Comment (DarinkaGM) [reply react/week1/my-app-w1/src/App.js:82]: I hadn't notice it changed! Not sure if it auto-formatted on my editor or here

File: react/week1/my-app-w1/src/App.js (hunk 1 above)
Comment (DarinkaGM) [reply react/week1/my-app-w1/src/App.js:96]: It seems to be working now! Thanks a lot for your feedback.
And I'm sorry I used the template but I just discovered Ant design and wanted to play around with their components.

**Review by senner008 — APPROVED** [review]

**Pull request merged**